docs/rancher/resource-quota.md

```diff
@@ -0,0 +1,55 @@
+1. Find the cluster and click **Projects/Namespaces** > **Create Project**.
+1. Enter the project **Name**. Then click the **Resource Quotas** tab and the **Add Resource** button. In the ** Resource Type** field, select `CPU Limit` or `Memory Limit` and set the **Project Limit** and **Namespace Default Limit** values, and click `Create`.
+  ![](/img/v1.2/rancher/create-project.png)
+You can adjust the limits as follows: 
+1. Find the newly created project, and click **Create Namespace**.
+1. Enter the namespace **Name** and adjust the limits. When the settings are done, click **Create**.
+  ![](/img/v1.2/rancher/create-namespace.png)
+
+## Overhead memory of virtual machine
+When you create a virtual machine, the virtual machine controller automatically adds overhead resources to the virtual machine. The overhead resources are used to ensure the regular operation of the virtual machine, and the configuration of memory limits requires more memory reservation.
+
+For more information, see [Memory Overhead](https://kubevirt.io/user-guide/virtual_machines/virtual_hardware/#memory-overhead).
+
+For more information on how to calculate memory overhead, see this [code](https://github.com/kubevirt/kubevirt/blob/v0.54.0/pkg/virt-controller/services/template.go#L1804).
+
+
+
+## Adjust ResourceQuota automatically during the migration
+When the resource quota defined by the `ResourceQuota` object is exhausted, you cannot migrate the virtual machine. During the migration, a pod with the same resources as the source VM will be automatically created. If the resources required for creating the pod exceed the quota configured, the migration cannot proceed.
+
+In Harvester, before the migration, the values of `ResourceQuota` are automatically increased based on the resources required by the target virtual machine. After the migration, `ResourceQuotas` are restored (reduced) to their previous values.
+
+Keep the considerations below in mind about automatically adjusting `ResourceQuota`:
+
+- When the value of `ResourceQuota` is being increased, if you create, start, or restore other VMs, Harvester will verify if the resources are sufficient based on the original `ResourceQuota`. If not, the system will prompt that the migration cannot be completed.
+- `ResourceQuota` cannot be changed during VM migration.
+- After `ResourceQuota` is increased, there may be resource competition between non-VM pods and VM pods, resulting in migration failure. Therefore, we don't recommend deploying hybrid pods in the same namespace.
+- Due to the webhook validator's unreliability, the VM Controller performs a second validation to verify resource sufficiency. If the resources are insufficient, the `RunStrategy` of the VM will become **Halted**, and the annotation `harvesterhci.io/insufficient-resource-quota: xxxxx` will be added to the VM custom resource, informing you that the VM was shut down due to insufficient resources.
```

Comment:
```suggestion
- Due to the concurrent limitation of the webhook validator, the VM controller will execute a secondary validation to confirm resource sufficiency. If the resources are insufficient, it will auto config VM's`RunStrategy` to **Halted**, and a new annotation `harvesterhci.io/insufficient-resource-quota: xxxxx` will be added to the VM object, informing you that the VM was shut down due to insufficient resources.
```